app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,38 @@
+class RecipeFoodsController < ApplicationController
+  def index
+    @recipe_foods = RecipeFood.all
+  end
+
+  def new
+    @foods = current_user.foods
+  end
```

Comment:
I am not sure if we should use `current_user.foods`. I think that maybe we should use `Food.all`